sjb/generate.py

```diff
@@ -258,6 +258,7 @@ def parse_action(action):
     debug("[INFO] Marking this as a {} job for the {} repo".format(action, target_repo))
 
 DEFAULT_DESCRIPTION = "<div style=\"font-size: 32px; line-height: 1.5em; background-color: yellow; padding: 5px;\">" + \
```

Comment:
```suggestion
DEFAULT_DESCRIPTION = "<div style=\"font-size: 32px; line-height: 1.5em; background-color: red; padding: 5px;\">*** THIS SERVER WILL BE SHUTDOWN AUGUST 2022 ***</div><div style=\"font-size: 16px; line-height: 1.5em; background-color: yellow; padding: 5px;\">" + \
```

Reply:
Mostly the same, just break out into another div.

Reply:
thanks
![Screen Shot 2022-04-21 at 6 38 30 PM](https://user-images.githubusercontent.com/93946516/164562974-5823267b-b846-4f2b-b303-baaac593827e.png)